data/lua/PKMN_RB/pkmn_rb.lua

```diff
@@ -148,7 +135,7 @@ function receive()
         curstate = STATE_UNINITIALIZED
         return
     elseif e == 'timeout' then
-        --print("timeout") -- this keeps happening for some reason? just hide it
+        print("timeout")
```

Comment:
May want to remove (or comment out) this print from your production lua.

Reply:
Actually, noticed a few other `print`s. Out of curiosity, how often does these status `print`s actually output to the console?

Reply:
This should only print when there's an actual timeout happening? It was commented out before because it was erroneously printing this message when I had it checking for messages received every frame. It shouldn't be printing anything after everything is finished connecting, until it is disconnected or something goes wrong. Most of the print messages, along with much of the test of this lua, are taken directly from the ff1 connector lua